src/components/App/App.js

```diff
@@ -33,7 +64,14 @@ export default class App extends Component {
             <h2 className='tagline'><i>find your next worst movie...</i></h2>
           </div>
         </header>
-        {!this.state.selectedMovie ? <div className='poster-container'>{this.state.movies.map(movie => <Poster key={movie.id} data={movie} fetchSingleMovie={this.fetchSingleMovie}/>)}</div> : <Movie data={this.state.selectedMovie} clearSelectedMovie={this.clearSelectedMovie} />}
+        {!this.state.selectedMovie ? <div className='poster-container'>
+          {this.state.movies.map(movie => 
+          <Poster key={movie.id} data={movie} error={this.state.singleMovieError} fetchSingleMovie={this.fetchSingleMovie}/>)}
+          </div> : 
+          <Movie data={this.state.selectedMovie} clearSelectedMovie={this.clearSelectedMovie} videos={this.state.selectedMovieVideos}/>}
+          <footer>
```

Comment:
I think we should conditionally render the whole footer section instead of just the text inside